.gitignore

```diff
@@ -22,7 +22,7 @@ DerivedData
 # Bundler
 .bundle
 
-Carthage
+# Carthage
```

Comment:
Should we remove this line instead of comment it out?

Reply:
I added a comment explaining why we made the change instead. Does that work for you?

Reply:
Nice! that's even better